react/week1/my-app/src/App.js

```diff
@@ -0,0 +1,37 @@
+//import logo from './logo.svg';
+import './App.css';
+import Header from "./Header.js";
+import ListItems from "./List.js";
+
+const toDoArray =[
+  {
+  description : "Clean the house",
+  dueDate: new Date(2020, 12, 25)
+},
+{
+  description : "Buy a new shirt",
+  dueDate: new Date(2021, 10, 20)
+},
+{description : "Finish my homework",
+  dueDate: new Date(2021, 5, 10)
+}
+]
+
+
+function HeaderWithList(){
+  return(
+    <div>
+    <Header title="To do list:" />
+    <ListItems list={toDoArray} />
+    </div>
```

Comment:
You could use [`React.Fragment`](https://reactjs.org/docs/fragments.html) or the [new syntax](https://reactjs.org/docs/fragments.html#short-syntax) `<> your code </>` instead of an extra wrapper `<div>`. This way you don't have an obsolete element in your markup.